app/assets/stylesheets/open_project_global/_variables.sass

```diff
@@ -194,3 +194,9 @@ $user-avatar-mini-border-radius:                50px !default
 $user-avatar-mini-width:                        20px !default
 
 $select-element-padding:                        3px, 24px, 3px, 3px
+
+$homescreen-content-bg-color:                   white
+$homescreen-block-bg-color:                     white
+$homescreen-footer-bg-color:                    #F8F8F8
+$homescreen-block-border-color:                 #EDEDED
+$homescreen-icon-color:                         #7B827B
```

Comment:
Those are the homescreen footer icons right?

Asking because there are other icons on the homescreen, receiving the font color of `#333`